Snapshot.android.js

```diff
@@ -0,0 +1,5 @@
+var Snapshot = {
+  get: 'Android not available yet'
```

Comment:
This should be a function that returns an error object to the supplied callback.